src/lib.rs

```diff
@@ -548,6 +547,10 @@ fn encode_block(fi: &FrameInvariants, fs: &mut FrameState, cw: &mut ContextWrite
         }
     }
 
+    if skip {
+        cw.bc.reset_skip_context(bo, bsize, xdec, ydec);
```

Comment:
This is already done on line 563.

Reply:
Oops, I had meant to move it there for local cohesion instead of just copying it.